### PR TITLE
🐧  Add torvalds/linux to projects for building bootconfig

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Add linux/tools/bootconfig to upstream projects. This project does not build linux but only builds a tool in the repo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: build/lib/common.sh [hunk 1]
```diff
@@ -284,12 +284,12 @@ function build::common::get_latest_eksa_asset_url() {
...
 function build::common::wait_for_tag() {
-  local -r tag=$1
+  local -r clone_url=$1
+  local -r tag=$2
   sleep_interval=20
   for i in {1..60}; do
     echo "Checking for tag ${tag}..."
-    git fetch --tags > /dev/null 2>&1
-    git rev-parse --verify --quiet "${tag}" && echo "Tag ${tag} exists!" && break
+    git ls-remote --tags --exit-code $clone_url "refs/tags/$tag" && echo "Tag ${tag} exists!" && break
```
Comment (jaxesn) [inline build/lib/common.sh:292]: changed this so we can call it before cloning to support cloning a specific tag

**Review by abhay-krishna — COMMENTED** [review]
/approve

**Pull request merged**